Packs/Trello/Integrations/Trello/Trello.yml

```diff
@@ -0,0 +1,520 @@
+  - arguments:
+    - default: false
+      description: Card name
+      isArray: false
+      name: name
+      required: true
+      secret: false
+    - default: false
+      description: Card Description - Markdown Compatible
+      isArray: false
+      name: desc
+      required: false
+      secret: false
+    - default: false
+      description: ID of list to create card within
+      isArray: false
+      name: list_id
+      required: true
+      secret: false
+    - default: false
+      description: CSV list of Labels (by ID) to add to the card. Use list-labels
+        command to display.
+      isArray: false
+      name: idLabels
+      required: false
+      secret: false
+    deprecated: false
+    description: Create a new card.
+    execution: false
+    name: create-card
+    outputs:
+    - contextPath: Trello.CreatedCard.Id
+      description: ID of created card
+      type: Unknown
+    - contextPath: Trello.CreatedCard.Name
+      description: Name of created card
+      type: Unknown
+    - contextPath: Trello.CreatedCard.Url
+      description: URL Of created card
+      type: Unknown
+    - contextPath: Trello.CreatedCard.IdList
+      description: ID of list this card belongs to
+      type: Unknown
+  - arguments:
+    - default: false
+      description: ID of card to be updated
+      isArray: false
+      name: card_id
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: If true, archives the card.
+      isArray: false
+      name: closed
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: CSV Of Trello Label IDs to add to card
+      isArray: false
+      name: idLabels
+      required: false
+      secret: false
+    - default: false
+      description: ID Of list to move card to.
+      isArray: false
+      name: idList
+      required: false
+      secret: false
+    deprecated: false
+    description: Update an existing card
+    execution: false
+    name: update-card
+    outputs:
+    - contextPath: Trello.UpdatedCard.Id
+      description: ID of Updated Card
+      type: Unknown
+    - contextPath: Trello.UpdatedCard.Name
+      description: Name of updated card
+      type: Unknown
+  - arguments:
+    - default: false
+      description: ID of card to delete
+      isArray: false
+      name: card_id
+      required: false
+      secret: false
+    deprecated: false
+    description: Delete a card
+    execution: false
+    name: delete-card
+  - arguments:
+    - default: false
+      description: 'Optional - the ID of  the board to query, if not provided will
+        use configured '
+      isArray: false
+      name: board_id
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: CSV of action types used to filter response.
+      isArray: false
+      name: filter
+      predefined:
```

Comment:
Something seems off with the predefined values here:
1. the first one is an empty value 
2. the rest have `|\n` - is it intentional ?

Reply:
Fixed - came across incorrectly